src/commands/whoami.rs

```diff
@@ -2,11 +2,17 @@ use crate::settings::global_user::GlobalUser;
 use crate::terminal::{emoji, message};
 
 pub fn whoami(user: &GlobalUser) {
-    let msg = format!(
-        "{} You are logged with the email '{}'.",
-        emoji::WAVING,
-        user.email
-    );
+    // Only returns email if present (using email + API key for auth).
```

Comment:
what happens with `whoami` if people use api tokens? should probably ping API with token and pull down email

Reply:
i can do that! will take a few mins to set up all the api handling.

Reply:
poop. the endpoint for getting this info about a user does not support API tokens yet. I have started another branch https://github.com/cloudflare/wrangler/tree/gabbi/add-api-token-whoami that we can merge in once this endpoint supports API tokens.